src/app/js/data/request-diff.json

```diff
@@ -0,0 +1,15 @@
.description) {\n                        $('#map-description').text(data.description);\n                    }\n                    $('#card').show();\n                }\n            });\n            $('#card-hide').click(function() {\n                $('#card-content').hide();\n                $('#card-hide').hide();\n                $('#card-show').show();\n            });\n            $('#card-show').click(function() {\n                $('#card-content').show();\n                $('#card-hide').show();\n                $('#card-show').hide();\n            });\n        });\n    </script>\n    <script>\n        var map = new ol.Map({\n            target: 'map',\n            view: new ol.View({\n                center: [0, 0],\n                zoom: 2\n            })\n        });\n\n        function setListeners() {\n            const layers = map.getLayers().getArray();\n\n            var swipe = document.getElementById('swipe');\n\n            layers[1].on('precompose', function(event) {\n                var ctx = event.context;\n                var width = ctx.canvas.width * swipe.value / 100;\n\n                ctx.save();\n                ctx.beginPath();\n                ctx.rect(0, 0, width, ctx.canvas.height);\n                ctx.clip();\n            });\n\n            layers[1].on('postcompose', function(event) {\n                var ctx = event.context;\n                ctx.restore();\n            });\n\n            layers[2].on('precompose', function(event) {\n                var ctx = event.context;\n                var width = ctx.canvas.width * swipe.value / 100;\n\n                ctx.save();\n                ctx.beginPath();\n                ctx.rect(width, 0, ctx.canvas.width - width, ctx.canvas.height);\n                ctx.clip();\n            });\n\n            layers[2].on('postcompose', function(event) {\n                var ctx = event.context;\n                const width = ctx.canvas.width * swipe.value / 100;\n\n                ctx.restore();\n\n                ctx.save();\n                ctx.fillStyle = '#ffffff';\n                ctx.beginPath();\n                ctx.fillRect(width - 6, 0, 12, ctx.canvas.height);\n                ctx.restore();\n            });\n\n            swipe.addEventListener('input', function() {\n                map.render();\n            }, false);\n        }\n\n        $.getJSON('tile.json', function(data) {\n            if (data.length !== 3) {\n                return;\n            }\n            $('#layer-left').text(data[1].name);\n            $('#layer-right').text(data[2].name);\n            for (var i = 0; i < data.length; i++) {\n                var tileJson = data[i];\n                var layer = new ol.layer.Tile({\n                    source: new ol.source.TileJSON({\n                        tileJSON: tileJson,\n                    })\n                });\n                map.addLayer(layer);\n            }\n            setListeners();\n        });\n    </script>\n</body>\n</html>"
```

Comment:
Just curious what you are using to serialize the HTML.

Reply:
I do 2 things: 
1. Replace all line breaks with `\n`
2. Replace all `"`s with `\"`
They are both so that it can be a valid JSON string, so maybe I could use a library to sterilize it?